ditto/src/main/java/eu/xenit/testing/ditto/internal/record/RecordLog.java

```diff
@@ -21,10 +30,18 @@ public RecordLog() {
      * this like a series of version control commits, starting from scratch.
```

Comment:
not really this PR, but I was reading the docs of this function and I think there is a typo

It should be `you can think of this`, instead of `you can this of this`?